### ISSUE TITLE
Panel Row not respecting dimensions and scroll argument in PyScript

### ISSUE THREAD

**Issue by owenlamont** [holoviz/panel#4597]
#### ALL software version info
* PyScript 2023.03.1
* Bokeh 2.4.3
* Panel 0.14.4
* Pyodide 0.22.1
* Browser: Vivaldi 5.7.2921.65 (Stable channel) (64-bit) 

#### Description of expected behavior and the observed behavior

Expect to see a small corner of this plot - constrained to 400px by 400px and scroll bars to move around the plot (This works as expected running Panel 0.14.4 and Bokeh 2.4.3 in Jupyter or through the Bokeh server). In the PyScript page though I see the full plot size and no scroll bars.

#### Complete, minimal, self-contained example code that reproduces the issue

```html
<html>
    <head>
        <title>Panel Example</title>
        <meta charset="iso-8859-1" />
        <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-2.4.3.js"
        ></script>
        <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-2.4.3.min.js"
        ></script>
        <script
            type="text/javascript"
            src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-2.4.3.min.js"
        ></script>
        <script
            type="text/javascript"
            src="https://cdn.jsdelivr.net/npm/@holoviz/panel@0.14.4/dist/panel.min.js"
        ></script>
        <link
            rel="stylesheet"
            href="https://pyscript.net/releases/2023.03.1/pyscript.css"
        />
        <script defer src="https://pyscript.net/releases/2023.03.1/pyscript.js"></script>
    </head>
    <body>
        <py-config>
            packages = [
                "https://cdn.holoviz.org/panel/0.14.4/dist/wheels/bokeh-2.4.3-py3-none-any.whl",
                "https://cdn.holoviz.org/panel/0.14.4/dist/wheels/panel-0.14.4-py3-none-any.whl",
                "matplotlib",
                "numpy"
            ]
        </py-config>

        <py-script>
            import panel as pn
            import matplotlib.pyplot as plt
            from panel.pane import Matplotlib

            fig, ax = plt.subplots(figsize=(10, 10))
            mpl_pane = Matplotlib(fig, dpi=200)
            pn.Row(
                mpl_pane,
                scroll=True, width=400, height=400
            ).servable(target='simple_app')
        </py-script>

        <div id="simple_app"></div>
    </body>
</html>
```

#### Screenshots or screencasts of the bug in action
Actual behaviour
![image](https://user-images.githubusercontent.com/12672027/229985852-93489fa6-be77-44e7-980f-8e13f0a1e6a1.png)

Expected behaviour (from Jupyter)
![image](https://user-images.githubusercontent.com/12672027/229985650-c4f5cf5f-78b4-4876-8108-4c1515208c6d.png)

**Comment by owenlamont** [comment]
I raised this for Row but Column has exactly the same problem.

**Comment by owenlamont** [comment]
I've also noticed the loading property for the busy spinner isn't working in PyScript, not sure if I should raise that as a separate issue or wait to see if it is resolved in Panel 1.0

**Comment by philippjfr** [comment]
Just tested it and both are working in current dev builds. I'll make an RC release to test available tomorrow.
